homeassistant/components/ring/event.py

```diff
@@ -0,0 +1,113 @@
+"""Component providing support for ring events."""
+
+from dataclasses import dataclass
```

Comment:
Please move the new platform into a separate PR

Reply:
Done in https://github.com/home-assistant/core/pull/125506